### ISSUE TITLE
Config options for Atomic Disassembler

### ISSUE THREAD

**Issue by DeliciousTea** [mekanism/Mekanism#5481]
The atomic disassembler does not appear to be coded like a regular tool or weapon and has no config options beyond energy use. At the very least, it'd be nice to see config options for damage, as having a multi tool that doesn't break, has high energy efficiency, and is one of the best weapons in the game all at once, is difficult to balance in most mod packs. I've tried adjusting its damage manually with mods like Material Changer and that doesn't even work due to the way it seems to be coded.

TLDR: Please add config options for the atomic disassembler's damage.

Thanks!

**Comment by pupnewfster** [comment]
Version [9.8.0](https://www.curseforge.com/minecraft/mc-mods/mekanism/files/2746197) was just released and should sync to the twitch client within the next couple hours.
